frameworks/cassandra/universe/config.json

```diff
@@ -55,6 +55,11 @@
           "description": "Enable DC/OS Virtual Networking",
           "type": "boolean",
           "default": false
+        },
+        "log_level": {
```

Comment:
This should have an enforcement that restricts you to the strings logback actually understands (ERROR, DEBUG, WARN, INFO, TRACE?)

Also, should we only allow DEBUG and INFO? Customers might elect for a more restrictive log level than we want them to have.